### DISCUSSION TITLE
Error pushing, now everything is up-to-date, but not appearing on github

### DISCUSSION THREAD

**Discussion by sroener** [community/community#54947] [Repositories]
### Select Topic Area

Question

### Body

Hi,

I tried to push some commits to one of my repositories. First, I got the following error:

```bash
remote: Resolving deltas: 100% (20/20), completed with 7 local objects.
remote: fatal error in commit_refs
To REPONAME.git
 ! [remote rejected] develop -> develop (failure)
error:  failed to push some refs to REPONAME
```

After another try it said everything is up-to-date, but the changes (new files) and commit history (several are missing) are not showing on GitHub.

Will these changes appear after they resolved the problems with the ungoing problems? Can I do something to push the changes again?

**Comment by AlbertShown** [comment]
I have the same issue. Let's wait while they recover push operations.

https://twitter.com/githubstatus/status/1655924259728891904
https://downdetector.ca/status/github/
https://www.githubstatus.com/

**Comment by sroener** [comment]
Update: 

Pulling the repo and checking out the branch in a different location does show the changes. After pushing another commit, the GitHub repo page also caught up on the changes.

**Reply by AlbertShown** [reply]
So, do we need to re-push?

**Reply by AlbertShown** [reply]
It's working, thanks.
